services/venice-controller/src/main/java/com/linkedin/venice/controller/server/AdminSparkServer.java

```diff
@@ -251,7 +251,7 @@ public boolean startInner() throws Exception {
       if (stats == null) {
         stats = nonclusterSpecificStats;
       }
-      stats.recordRequest();
+      stats.recordRequest(request.url());
```

Comment:
1. can we use ControllerRoute.valueOfPath(request.uri())) here instead of url? is there any specific need for the URL here? maybe like needing request.query_string?
2. Do we also need to add HttpMethod here?
3. Do we get the caller details here? based on the SSL handling at line 235?